### DISCUSSION TITLE
Multiple Sass Files

### DISCUSSION THREAD

**Discussion by michaeld96** [Appsilon/rhino#318] [Q&A] [answered]
Is it possible to have multiple Sass files inside of a Rhino project? If so, are they made with `rhino::build_sass()`?

**Comment by kamilzyla** [comment, marked as answer]
Yes, you can have any file/directory structure in `app/styles` that you desire. The `app/styles/main.scss` is the entrypoint: you'll need to [`@use`](https://sass-lang.com/documentation/at-rules/use) the other files from `main.scss`. Then running `rhino::build_sass()` will be sufficient to include the styles in your application.
